### PR TITLE
Change pagination btns to anchors & fix tabindices

### PR DESCRIPTION
**Pull request by alshakero**
Fix https://github.com/Starcounter/KitchenSink/issues/263.

Needs https://github.com/Starcounter/uniform.css/pull/82

### CODE REVIEW

File: src/KitchenSink/wwwroot/KitchenSink/PaginationPage.html [hunk 1]
```diff
@@ -24,20 +24,23 @@ <h1 slot="kitchensink/pagination-heading">Pagination</h1>
             </template>
             </tbody>
         </table>
-        <span slot="kitchensink/pagination-navigation-first" tabindex="0" value="{{model.FirstPageTrigger$::click}}" on-click="onFirstPageClick" href="">First</span>
-        <span slot="kitchensink/pagination-navigation-previous" disabled$="{{model.DisableFirst}}" tabindex="0" value="{{model.PreviousPageTrigger$::click}}">Previous</span>
+        <a slot="kitchensink/pagination-navigation-first" tabindex="0" value="{{model.FirstPageTrigger$::click}}" on-click="onFirstPageClick" href="">First</a>
+        <a slot="kitchensink/pagination-navigation-previous" disabled$="{{model.DisableFirst}}" tabindex="1" value="{{model.PreviousPageTrigger$::click}}">Previous</a>
```
[the remark this answers is not in the document]
Comment (alshakero) [reply src/KitchenSink/wwwroot/KitchenSink/PaginationPage.html]: Good point.

[the remark this answers is not in the document]
Comment (tomalec) [reply src/KitchenSink/wwwroot/KitchenSink/PaginationPage.html]: It seems browser defaults are not enough if there is no `href` so we need either href or tabindex.

**Comment by tomalec** [comment]
Do I have something missing or borders are corrupted?
image

Also, I see no outline when I tab through the page links (after I set `href` or `tabindex="0"`)

**Comment by tomalec** [comment]
Now element is focusable, but I cannot see any visual indication when it's focused, do I miss something?

**Comment by alshakero** [comment]
Now looks and works perfect in FF and Chrome. Requires https://github.com/Starcounter/uniform.css/pull/92

**Pull request merged**